### DISCUSSION TITLE
New Official Release?

### DISCUSSION THREAD

**Discussion by friedpa** [jomjol/AI-on-the-edge-device#3189] [General]
Dear Developers,
there has been a lot of new functionallity added in the last couple of months.
I think it is time to test a further official mster and release it as soon as possible.
Thanks!

**Comment by caco3** [comment]
It would be of great help if you grab rolling and see if everything still works for you.

We are just in the progress of planning a new release

**Comment by friedpa** [comment]
I am on the latest rolling (also tested the WiFi configuration from @SybexX)
So far not any problems.

**Comment by caco3** [comment]
See https://github.com/jomjol/AI-on-the-edge-device/discussions/3246
